### ISSUE TITLE
Not able to reproduce the results

### ISSUE THREAD

**Issue by OmkarThawakar** [wjf5203/SeqFormer#9]
Congrats for the awesome work. 

I am trying to reproduce the results for resnet-50 backbone. 
I tried following  ,
1. Train Seqformer on coco dataset (with num_frames=1) for 24 epochs
2. Train Seqformer on coco+ytvis and ytvis using coco pretrained weights

Still I am not able to generate the desired numbers. 

Can you please help me out with this ?

Thanks,

**Comment by wjf5203** [comment]
Hi, thanks for your attention.
We pretrain Seqformer for 50 epochs on coco in the first step, following DeformableDETR.
It is recommended to use the coco pre-trained models we provide in the model zoo.

**Comment by chrisjuniorli** [comment]
Hi, I'm also working on reproducing the ResNet-50 based result.
I pretrained SeqFormer for 50 epochs on COCO following Deformable DETR, here is the training config I used:

python -u main.py \
    --dataset_file coco \
    --epochs 50 \
    --lr 2e-4 \
    --lr_drop 40 \
    --batch_size 2 \
    --num_workers 2 \
    --coco_path ../data/COCO \
    --ytvis_path ../data/Youtube-VOS/2019 \
    --num_queries 300 \
    --enc_layers 6 \
    --dec_layers 6 \
    --num_frames 1 \
    --with_box_refine \
    --masks \
    --rel_coord \
    --backbone resnet50 \
    --output_dir outputs/seqformer_coco \

Then I fine-tuned it on Youtube-VIS following r50_seqformer_ablation.sh and got a 42.1 mAP, which is about ~3% mAP lower than the 45.1 mAP reported on the ablation model zoo, could you help point out the difference between my pretraining setting to yours?

@wjf5203
